tests/python/relay/test_backend_graph_executor.py

```diff
@@ -133,7 +133,7 @@ def test_plan_memory():
     storage_ids = set()
     device_types = set()
     for k, v in smap.items():
-        assert len(v) == 2
+        assert len(v) == 3
```

Comment:
want to assert on the v[2] element?

Reply:
Also here, I am not sure exactly what you mean

Reply:
here you're expanding the length assertion, but below, there are some rudimentary asserts on v[0] and v[1]. can you also add some assert on v[2]?